### ISSUE TITLE
Unable to install toolchain 1.55

### ISSUE THREAD

**Issue by rnbguy** [foresterre/cargo-msrv#94]
The issue comes because `rust-releases` parses the Rust [`RELEASES.md`][rust-release] without looking at the release date if it is in the future or not.

The current [`RELEASES.md`][rust-release] starts like this,
```
Version 1.55.0 (2021-09-09)
============================
...
```

and [`rust-releases-rust-changelog/src/lib.rs#L42-L49`][rust-changelog] has this,
```
        let releases = content
            .lines()
            .filter(|s| s.starts_with("Version"))
            .filter_map(|s| {
                s.split_ascii_whitespace()
                    .nth(1)
                    .and_then(|s| semver::Version::parse(s).map(Release::new_stable).ok())
            });
```

PS. A workaround is to add `--max 1.54.0` until `1.55.0` releases.

[rust-release]: https://raw.githubusercontent.com/rust-lang/rust/master/RELEASES.md
[rust-changelog]: https://github.com/foresterre/rust-releases/blob/32f2c20cbdf8016b79b5dd8f36d410bfc991cd21/crates/rust-releases-rust-changelog/src/lib.rs#L42-L49

**Comment by foresterre** [comment]
Thanks for your report 😄! This will be fixed in https://github.com/foresterre/rust-releases/pull/46. The dependency will be updated to the latest `rust-releases` soon after this PR has been merged.

**Comment by rnbguy** [comment]
Awesome! :) yea, I didn't create the issue at `rust-releases` because it looked like it is `cargo-msrv` specific and you are the owner of both repos.

**Comment by foresterre** [comment]
Closed by #95 and https://github.com/foresterre/rust-releases/pull/46.
